pints/functionaltests/_problems.py

```diff
@@ -0,0 +1,203 @@
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+
+import numpy as np
+import pints
+import pints.toy
+
+
+class RunMcmcMethodOnProblem(object):
```

Comment:
I suppose that if these methods are going to live in the main PINTS repo they will need to have full docstrings and be listed in the docs etc?

Reply:
Placing them inside the `pints` module also means that they'll get distributed as part of the package. Is that something we want?